install/zitadel-install.sh

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2025 community-scripts ORG
+# Author: dave-yap
+# License: MIT | https://github.com/community-scripts/ProxmoxVE/raw/main/LICENSE
+
+source /dev/stdin <<<"$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies (Patience)"
+$STD apt-get install -y \
+    curl \
+    ca-certificates \
+    wget \
+    sed \
+    sudo
+msg_ok "Installed Dependecies"
+
+msg_info "Installing Postgresql"
+$STD apt-get install -y postgresql postgresql-common
+sudo /usr/share/postgresql-common/pgdg/apt.postgresql.org.sh <<< return &>/dev/null
+DB_NAME="zitadel"
+DB_USER="zitadel"
+DB_PASS="zitadel"
```

Comment:
Generate password like this
```suggestion
DB_PASS=$(openssl rand -base64 18 | tr -dc 'a-zA-Z0-9' | cut -c1-13)
```

Reply:
Not sure how to implement this into the config.yaml but will have a look!

Reply:
> Not sure how to implement this into the config.yaml but will have a look!

Here an example how I did it in baikal script
```
cat <<EOF >/opt/baikal/config/baikal.yaml
database:
    backend: pgsql
    pgsql_host: localhost
    pgsql_dbname: $DB_NAME
    pgsql_username: $DB_USER
    pgsql_password: $DB_PASS
EOF
```

Reply:
Looks good, implementing now!